listenbrainz/spark/handlers.py

```diff
@@ -190,20 +190,22 @@ def handle_sitewide_entity(data):
 def handle_dump_imported(data):
     """ Process the response that the cluster sends after importing a new dump
 
-    We don't really need to _do_ anything, just send an email over for observability.
+    We don't really need to _do_ anything, just send an email over if there was an error.
     """
     if current_app.config['TESTING']:
         return
 
-    dump_name = data['imported_dump']
+    errors = data['errors']
     import_completion_time = data['time']
-    send_mail(
-        subject='A {} has been imported into the Spark cluster'.format(' '.join(data['type'].split('_')[1:])),
-        text=render_template('emails/dump_import_notification.txt', dump_name=", ".join(dump_name), time=import_completion_time),
```

Comment:
Is this template file still in use? If not, we should remove it in this PR.

Reply:
Removed.